### PR TITLE
77 move functionality to pipelines

### PR DESCRIPTION
**Pull request by KwiatkowskiML**

### CODE REVIEW

**Review by Jlisowskyy — APPROVED** [review]
LGTM, but needs another review at least

**Review by KwiatkowskiML — COMMENTED** [review]
looks alright

File: src/test/test_classify.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import numpy as np
+import soundfile as sf
+
+from src.cnn.cnn import BasicCNN
+from src.pipeline.audio_data import AudioData
+
+# pylint: disable=line-too-long
+TEST_FILE_PATH = str(Path.resolve(Path(f'{__file__}/../test_data/f2_script1_ipad_office1_35000.wav')))
+MODEL_PATH = "/home/michal/studia/sem5/ml/intro-ml-2024/models/model.pth"
```
Comment (mycielskit) [inline src/test/test_classify.py:17]: *kaszl*

**Review by mycielskit — CHANGES_REQUESTED** [review]

**Review by mycielskit — APPROVED** [review]
Let's just keep that in mind for the next commit

**Pull request merged**